### PR TITLE
[SVCS-675] Add file size limit to tabular renderer.

### PR DESCRIPTION
**Pull request by felliott**
## Ticket

*Not created yet*

## Purpose

Refuse to render tabular files if they exceed a certain size.

## Changes

Add a configurable setting for tabular file size limits.  If files is above this size, refuse to render it.

### CODE REVIEW

File: mfr/extensions/tabular/exceptions.py [hunk 1]
```diff
@@ -42,3 +47,20 @@ def __init__(self, message, *args, code: int=500, formatting_function: str='', *
         super().__init__(message, *args, code=code, **kwargs)
         self.formatting_function = formatting_function
         self.attr_stack.append([self.__TYPE, {'formatting_function': self.formatting_function}])
+
+
+class FileTooLargeError(TabularRendererError):
```
Comment (cslzchen) [inline mfr/extensions/tabular/exceptions.py]: I remember you mentioned in WB but I am not sure about MFR: when an exception is added, should we add it to somewhere in tests to make sure  thatserialization works?

File: mfr/extensions/tabular/render.py [hunk 2]
```diff
@@ -1,12 +1,16 @@
-import os
 import json
+import logging
+import os
```
[the remark this answers is not in the document]
Comment (felliott) [reply mfr/extensions/tabular/render.py]: I thought for MFR we were doing alphabetical order, or am I misremembering that?

File: mfr/extensions/tabular/render.py [hunk 3]
```diff
@@ -1,12 +1,16 @@
...
 
+from humanfriendly import format_size
 from mako.lookup import TemplateLookup
-from mfr.core import extension
 
+from mfr.core import extension
 from mfr.extensions.tabular import settings
 from mfr.extensions.tabular import exceptions
```
[the remark this answers is not in the document]
Comment (felliott) [reply mfr/extensions/tabular/render.py]: :+1:

File: mfr/extensions/tabular/exceptions.py (hunk 1 above)
Comment (felliott) [reply mfr/extensions/tabular/exceptions.py]: Not needed for MFR.  That test is to make sure the error survives the pickling/unpickling done by WB's celery worker.

**Pull request closed without merging**